DownFall/Extensions/CGPoint+Extension.swift

```diff
@@ -34,4 +35,132 @@ extension CGPoint {
         
         return points
     }
+    
+    static func positionThis(_ this: CGRect,
+                             inTopRightOf that: CGRect,
+                             padding: CGFloat = Style.Padding.normal) -> CGPoint {
+        return CGPoint(x: that.width/2 - this.width/2 - padding,
+                       y: that.height/2 - this.height/2 - padding)
+    }
+    
```

Comment:
This might be worth thinking more about.  It seems like objects want to positions relative to each other in a couple of ways.